packages/buendia-backup/data/usr/bin/buendia-db-is-unused

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2019 The Project Buendia Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distrib-
+# uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+# specific language governing permissions and limitations under the License.
+
+if [ "$1" = "-h" ]; then
```

Comment:
Naming this script (and the underlying concept) is tricky.  "Unused" to me suggests a resource not in use, which doesn't seem right — the currently running OpenMRS process does have it open for reading and writing, is displaying the contents of this database and will write to it.  "Unused" to me would indicate that the OpenMRS process is currently not running.

The thing we're looking for is "contains no useful data".  So far "pristine" is the best I can come up with (conveying "in its original state").  Also maybe possible is "empty", as it would look empty from a user perspective, though that could be confusable with "empty" meaning a database that has no tables or no rows.

Reply:
Agreed, but "pristine" seems kind of odd to me too. I'm opting for "empty".